infrastructure/utils.py

```diff
@@ -103,10 +105,15 @@ def load_csv(geography, fp):
 
 
 @transaction.atomic
-def load_file(geography, reader, financial_year):
+def load_file(geography, reader, financial_year=None):
     print(geography.geo_code)
 
     for idx, row in enumerate(reader):
+        #skip columns // Need to be replace with Proper data
+
+        if idx + 2 == 2:
+            break
```

Comment:
I don't think we should have this any more? This was for the wrong files - I think this might break the files at https://drive.google.com/drive/u/0/folders/1Ry360v9ZMPpGTlfjfhqMYcZ2oCdNeXiS